lib/cisco_node_utils/cmd_ref/acl.yaml

```diff
@@ -0,0 +1,29 @@
+# Command Reference Common ACL
+#
+# For documentation please see:
+#   - README_YAML.md
+#
+---
+_template:
+  config_get: 'show run aclmgr'
+  config_get_token: '/^<afi> access-list <acl_name>$/'
+  config_set: '<afi> access-list <acl_name>'
+
+acl:
+  config_get_token: '/^<afi> access-list (\S+)$/'
+  config_set: '<state> <afi> access-list <acl_name>'
+                      
+all_acl:
+  multiple:
+  config_get_token: '/^<afi> access-list (\S+)$/'
+
+fragments:
+  config_get_token_append: '/fragments permit-all$|fragments deny-all$/'
+  config_set_append: '<state> fragments <permit>'
+  default_value: nil
```

Comment:
Use `~` here instead of `nil`. 

https://github.com/saqibraza/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/README_YAML.md#default_value